src/components/header/DropdownMenu.jsx

```diff
@@ -58,14 +59,21 @@ export default function DropdownMenu({ open, authed }) {
             ) : null}
             <a
               href="/about"
-              className="text-gray-700 px-4 py-3 text-sm flex border-b hover:bg-gray-100 items-center "
+              className="text-gray-700 px-4 py-3 text-sm flex border-b hover:bg-gray-100 items-center"
               role="menuitem"
               tabIndex="-1"
               id="menu-item-2"
             >
               <AboutButton tw="mr-2" />
               <p className="mt-auto mb-auto">About Us</p>
-            </a>
+            </a>{" "}
```

Comment:
I think on desktop the search is fine in the corner. But on mobile, I think it should be in a more accessible / easy to see spot. Unfortunately we don't have much data regarding if people actually click our dropdown menu button, but if I had to guess it's quite low. 
![Screen Shot 2023-07-15 at 12 58 01 PM](https://github.com/alexsaltstein/HobokenEventsWebApp/assets/25209141/422a430c-d6a6-4df7-90b6-4d5e769fcb2f)

My suggestion is right above What's Happening. To be honest i think the NYC skyline image being there on mobile provides no real value besides stating our mission statement. If I'm coming to the site I want to see the deals and an easy an "in front of me" way (search bar) to find the places I'm looking at.